daemons/dss-notify-v2/Readme.md

```diff
@@ -50,6 +53,9 @@ Bundle creation notifications have the format
 The `jmespath_query` and `attachment` fields are subscription dependent and may not be present. Attachment
 usage and format description can be found in the /subscription endpoint of the [DSS OpenAPI specifications](../../dss-api.yml).
 
+The `event_timestamp` field is a timestamp for when the event was created, the formatting is in the `DSS_VERSION` format.
```

Comment:
Perhaps "The `event_timestamp` field is the event creation date in the `DSS_VERSION` format."